warehouse/query-core/src/main/java/datawave/query/transformer/UniqueTransform.java

```diff
@@ -42,13 +26,17 @@ public class UniqueTransform extends DocumentTransform.DefaultDocumentTransform
     
     private static final Logger log = Logger.getLogger(UniqueTransform.class);
     
+    private final UniqueUtil uniqueUtil = new UniqueUtil();
     private BloomFilter<byte[]> bloom;
     private UniqueFields uniqueFields;
     private Multimap<String,String> modelMapping;
+    private LinkedList<Entry<Key,Document>> documents;
     
-    public UniqueTransform(UniqueFields uniqueFields) {
-        this(null, uniqueFields);
-    }
+    /**
+     * how long (in milliseconds) to let a page of results to collect before signaling to return a blank page to the client (which indicates the request is
+     * still in process and all results will be returned at once at the end)
+     */
```

Comment:
I'm not in love with this wording, but I also don't have a great alternative. Took a stab below, feel free to improve it or just ignore this comment.

"Length in milliseconds that a client will wait while results are collected. If a full page is not collected before the timeout, a blank page will be returned to signal the request is still in progress."

Reply:
I took your suggestion and just changed it to "Length of time....". I also changed the comment in GroupingTransform since it was the same wording for the same variable declaration.